core/tee/tee_cryp_utl.c

```diff
@@ -16,6 +16,13 @@
 #include <trace.h>
 #include <utee_defines.h>
 
+static bool tee_cryp_inited;
```

Comment:
This is something internal to the DT driver probing. Could it be kept there instead?
`probe_dt_drivers_early()` and `probe_dt_drivers()` already depends on `tee_cryp_init()` being called somewhere in between so it could just as well be assumed that it has been called when `probe_dt_drivers()` is called.

Reply:
Would be ok but I feared _tee_cryp_util.c_ is modified, for example changing `tee_cryp_init()` from `service_init` to another initcall level. Do you think it's worth to assume it won't change?

Reply:
OK, so this is for debug purposes.
How about calling a `dt_driver_crypt_init_complete()` at the end of `tee_cryp_init()`? In case the DT driver framework isn't used it can do nothing else take some action in case asserts are enabled or something.

Reply:
+1, thanks for the suggestion.